testing/MLDB-2110-merge-and-subselect-progress.py

```diff
@@ -0,0 +1,61 @@
+# MLDB-2110-merge-and-subselect-progress.py
+# Datacratic, 2017
```

Comment:
Guy Dumais

Did you use the script generator? It should have put your name there, otherwise it's a bug.